src/main/java/seedu/mark/logic/parser/ParserUtil.java

```diff
@@ -100,6 +102,79 @@ public static Url parseUrl(String url) throws ParseException {
         return new Url(trimmedUrl);
     }
 
+    /**
+     * Parses a {@code String arg} into a {@code Tab}.
+     * Parsing will be successful only if {@code arg} is either "1", "2" or "3".
+     *
+     * @param arg The argument of a tab command
+     * @return The corresponding tab
+     * @throws ParseException if the given {@code arg} is invalid.
+     */
+    public static TabCommand.Tab parseTabIndex(String arg) throws ParseException {
+        Index index = null;
+        try {
+            index = ParserUtil.parseIndex(arg);
```

Comment:
I think... it might be easier to just copy the code over from `parseIndex` and modify it to get a `Tab` directly (instead of getting an `Index` first haha).

Reply:
Makes sense, I'll just `parseInt` directly